src/index.js

```diff
@@ -1,9 +1,8 @@
-import React from 'react';
+import { React, Suspense } from 'react';
```

Comment:
I don't think you can import react this way, right?
```js
import React, { Suspense } from 'react';
```

Reply:
Sorry for that, I need to read a bit more....it should be fixed now.

Reply:
Nothing to say sorry about. And, that is awesome 🚀